recipes/glue-qt/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "glue-qt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/glue-qt-{{ version }}.tar.gz
+  sha256: 26a7f10e8bb58426512921355a0b9078698170fcc66504b2174969e1d1073d42
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - glue = glue_qt.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  osx_is_app: true
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+  host:
+    - python
+    - setuptools-scm
+    - pip
+  run:
+
+    # The following is needed to make sure that the package works as a GUI
+    # application (glue needs to be run with python.app, not python)
+    - python.app  # [osx]
```

This is a trick to have an osx only dependency but keep it noarch.
See https://github.com/conda-forge/click-feedstock/blob/main/recipe/meta.yaml for an example.
(You'll probably need to adjust the build string too.)

```suggestion
    - python.app  # [osx]
    - __win      # [win]
    - __unix     # [unix]
```